SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -113,63 +128,68 @@ etc...
-    1. In File Explorer, right-click your uproject file and select **Generate Visual Studio project files**.
-    1. Open the solution in Visual Studio, and compile and run the project in the Development Editor configuration.
-
-## Running the Schema Generator and setting up the UE4 Editor
-1. In the Unreal Editor, on the SpatialOS Unreal GDK toolbar, click **Generate Schema** to run the `Schema Generator`.
-1. On the same toolbar, click **Snapshot**.
+Set your Unreal project to work with the SpatialOS GDK for Unreal fork of the Unreal Engine, which you cloned and installed earlier. To do this:
+1. In File Explorer, navigate to `<ProjectRoot>\<GameRoot>`.
+1. Right-click your `.uproject` file and select **Switch Unreal Engine version**.
+1. Select the path to the Unreal Engine fork you cloned earlier.
+1. In a terminal window, from `<ProjectRoot>\<GameRoot>\Plugins\UnrealGDK\SpatialGDK\Build\Scripts`, run `BuildWorker.bat <YourProject>Editor Win64 Development <YourProject>.uproject`   
+For Example "`BuildWorker.bat StarterProjectEditor Win64 Development StarterProject.uproject`"  
+This allows you to run locally managed workers. For example running a local spatial deployment with `one_worker` or `two_worker` launch configurations.
+1. In File Explorer, right-click your `.uproject` file and select **Generate Visual Studio project files**.
+1. Open the solution in Visual Studio, and compile and run the project in the Development Editor configuration.
+
+## Running the Schema and Snapshot Generator
+1. In the Unreal Editor, on the SpatialOS Unreal GDK toolbar, click  the **Schema** button to run the `Schema Generator`.
+1. On the same toolbar, click the **Snapshot** button.
 
 ## Running your game
-1. In the Unreal Editor, on the toolbar, open the **Play** drop-down menu and check the box next to **Run Dedicated Server**.
-1. On the SpatialOS Unreal GDK toolbar, click **Launch** and wait for the output `SpatialOS ready`.
+1. In the Unreal Editor, on the toolbar, open the **Play** drop-down menu and check the box next to **Run Dedicated Server** as well as the checkbox for **Spatial Networking**
+1. On the SpatialOS Unreal GDK toolbar, click the green **Launch** (not the default Launch button in the Unreal Editor), this will build your worker configurations (`spatial build build-config`) and `spatial local launch` will be ran in a separate console window, wait for the output "`SpatialOS ready`".
 1. On the main Unreal toolbar, click **Play**.
+
+## Modifying default behavior
+If you wish to modify the default behavior for the SpatialOS GDK for Unreal; such as the default snapshot save path and filename, the spatial launch configuration used when pressing **Launch** and more... you can modify these settings from the Unreal Editor by navigating to `Edit->Project Settings->SpatialOS Unreal GDK->Toolbar`. 
```

Comment:
No need to have the first sentence as it repeats the title.

REVISED:

"You can modify some of the GDK settings from the Unreal Editor toolbar at **Edit** > **Project Settings** >* *SpatialOS Unreal GDK** > **Toolbar**.
You can change:
*  the snapshot file's filename and location
*  the launch configuration